.ci/test.sh

```diff
@@ -16,3 +16,6 @@ mvn clean test -am -pl flink/flink-1.12
 mvn clean test -am -pl flink/flink-1.13
 mvn clean test -am -pl mapreduce/mapreduce-base
 mvn clean test -am -pl prestodb
+
+export JAVA_HOME=/home/jenkins/agent/lib/jdk-11.0.12
+mvn clean compile -am -pl prestosql
```

Comment:
shoule be `mvn test compile -am -pl prestosql ` ?

Reply:
fixed